frontend/admin/partners/editable-partner-service-row.component.tsx

```diff
@@ -61,9 +61,9 @@ export default function EditablePartnerServiceRow(
       </tr>
       {showingModal && (
         <Modal
-          headerText="Modify Partner"
+          headerText="Modify Service"
```

Comment:
👍  good catch. We have a similar modal for CU services, so let's clarify which kind of service this is.

```suggestion
          headerText="Modify Partner Service"
```